Tests/AK/TestHashTable.cpp

```diff
@@ -5,6 +5,7 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include "AK/Forward.h"
```

Comment:
Do we need this include? And if so, can you change it to `#include <AK/Forward.h>` to match the others?

Reply:
Ah I hate my clangd, adding includes without being asked to do so